andela/mmdp_site
# MMDP-315 - Implements frontend for adding a stakeholder

[Delivers MMDP-315]

**What does this PR do?**
This PR enables a user to add a stakeholder

**Description of Task to be completed?**

- a user should be able to navigate to the next and previous form
- validate form input

**How should this be manually tested?**
- Check out `ft-addStakeholder-FE-MMDP-315`
- Install dependencies
- Run both the client and server using `yarn run dev`
- Open  this link in your browser http://localhost:1234/stakeholder-directory 
- Click the `Add stakeholder button` to add a new stakeholder
- Click the `Next button` at the bottom of the form to add a beneficiary service
- Click the `Add new beneficiary button` at the bottom of the form to add a beneficiary service

**What are the relevant Jira issues?**

[MMDP-315](https://viisaus.atlassian.net/browse/MMDP-315)

**Screenshots (if appropriate)**
